Update enabling-404-pages.md (#642)
vtex-apps/io-documentation: docs/en/Recipes/store-management/enabling-404-pages.md

> # Enabling 404 pages

Author: ```suggestion
---
title: Enabling 404 pages
description: "Avoid problems with SEO by enabling 404 pages in your VTEX IO store."
date: "2020-05-23"
tags: ["404", "404-pages", "not-found", "http"]
version: "0.x"
git: "https://github.com/vtex-apps/io-documentation/blob/master/docs/en/Recipes/store-management/enabling-404-pages.md"
---

# Enabling 404 pages
```

For e-commerce companies, **one of the main advantages of enabling 404 pages is to avoid that Google indexes pages with paths that do not exist**, thereby harming the store's SEO and its customer experience.

In VTEX IO, 404 pages are **natively** enabled for inexistent product pages and for URLs that have more than one invalid segment. But in addition, it is also possible to allow 404 pages when URLs have a **single** invalid segment, as shown in the step by step section below.

<div class="alert alert-info">

> Due to implementation rules, the native functionality of 404 pages does not function for URLs originating from user searches. The reason behind this is that URLs originating from product searches include the <code>map=ft</code> parameter in their path, parameter which helps the platform to interpret that such URL segments reflect user performed searches, and thus should not be interpreted as non-existent, which would lead to a 404 page being rendered.

Reviewer: ```suggestion
Due to implementation rules, the native functionality of 404 pages does not function for URLs originating from user searches. The reason behind this is that URLs originating from product searches include the <code>map=ft</code> parameter in their path, helping the platform to interpret that such URL segments reflect user performed searches, and thus should not be interpreted as non-existent (which would lead to a 404 page being rendered).
```

<div>

<div class="alert alert-warning">

> In rare cases when the parameter <code>map=specificationFilter</code> is present in the path of such search originated URLs, 404 pages may be rendered. This happens because the platform cannot interpret these URL segments as originating from user searches, since it cannot find the map=ft parameter usually present in such URLs, and therefore renders a 404 error page. This behavior is uncommon and is being addressed by our product team.

Author: ```suggestion
In rare cases when the parameter <code>map=specificationFilter</code> is present in the path of such search originated URLs, 404 pages may be rendered. This happens because the platform cannot interpret these URL segments as originating from user searches, since it cannot find the <code>map=ft</code> parameter usually present in such URLs, and therefore renders a 404 error page. This behavior is uncommon and is being addressed by our product team.
```

> In rare cases when the parameter <code>map=specificationFilter</code> is present in the path of such search originated URLs, 404 pages may be rendered. This happens because the platform cannot interpret these URL segments as originating from user searches, since it cannot find the <code>map=ft</code> parameter usually present in such URLs, and therefore renders a 404 error page. This behavior is uncommon and is being addressed by our product team.

Reviewer: ```suggestion
In rare cases when the parameter <code>map=specificationFilter</code> is present in the path of such search originated URLs, 404 pages may be rendered. This happens because the platform cannot interpret these URL segments as originating from user searches since it cannot find the <code>map=ft</code> parameter usually present in such URLs. This behavior is uncommon and is being addressed by our product team.
```

![not-found-block](https://user-images.githubusercontent.com/52087100/76447318-4108b780-63a7-11ea-9b03-77413e0e4855.png)

> 5. If you're happy with all your configurations, repeat all of the block configuration in a [production workspace](https://vtex.io/docs/recipes/store/creating-a-production-workspace) this time.

Reviewer: ```suggestion
5. If you're happy with all configurations done in your Development workspace, it is time do [make your new theme version publicly available](https://vtex.io/docs/recipes/store-management/making-your-theme-content-public/). 
```

> 6. Lastly, [promote your production workspace](https://vtex.io/docs/recipes/store/promoting-a-workspace-to-master) to Master.

Reviewer: ```suggestion
```
